src/vm/excep.cpp

```diff
@@ -7784,11 +7783,19 @@ LONG WINAPI CLRVectoredExceptionHandler(PEXCEPTION_POINTERS pExceptionInfo)
     //
     return retVal;
 #else // !FEATURE_PAL
+
+#if defined(WIN64EXCEPTIONS) && defined(FEATURE_HIJACK)
```

Comment:
This doesn't look like what I meant. Is there a way to see the previous version somehow so that I can check what  was what I really was suggesting?